replicate.Rmd

```diff
@@ -0,0 +1,1684 @@
+#produces figure 5
+
+print(figure5)
+#displays figure 5
+```
+
+
+
+> The interaction between dieting status and condition was significant, F(1, 230) = 4.40, p < .05, ηp2 = .02,
+
+```{r echo = FALSE}
+# ANOVA : interaction effects
+# income treated as split : somewhat matches
+summary(aov(fattening ~ split * week, df5_spss))
+temp_eta2 <- etaSquared(aov(fattening ~ split * week, df5_spss)) %>%
+  .[3,2] %>%
+  round(digit=2)
+print(c("partial eta squared:", temp_eta2), quote=F)
+```
+
+> and the interaction was also found when dieting behavior was treated continuously, β = 0.43, t(230) = 2.11, p < .05.
+
+```{r echo = FALSE}
+
+lm(scale(fattening) ~ scale(log) + scale(week) + scale(log * week), data = df5_spss) %>%
+  tidy() %>%
+  filter(term == "scale(log * week)") %>%
+  mutate(beta = round(estimate, digit = 2),
+         p = round(p.value, digit = 4)) %>%
+  select(term, beta, p)
+
+```
+
+---
+
+### Study 6
+
+> In Study 6, there were 74 participants (mean age = 32.0 years; 46 females, 28 males).
+
+```{r echo = FALSE}
+suppressMessages(suppressWarnings(df6_demo <- read.csv("csv_files/6_demo.csv", header=T)))
+suppressMessages(suppressWarnings(df6_raw <- read.csv("csv_files/6_raw.csv", header=T)))
+suppressMessages(suppressWarnings(df6_cric <- read.csv("csv_files/6_cric.csv", header=T)))
+suppressMessages(suppressWarnings(df6_spss <- read.spss("framing/study_6.sav", to.data.frame = TRUE)))
+```
+
+```{r echo = FALSE}
+# Number of subjects
+df6_demo %>%
+  select(Subject) %>%
+  summarise(n())
+
+# Mean age
+df6_demo %>%
+  summarise(mean(Age, na.rm=T)) %>%
+  round(digit=1)
+
+# Number of females vs. males
+df6_demo %>%
+  group_by(Gender) %>%
+  summarise(n())
+```
+
+> One participant was excluded because of a computer malfunction during the game (not specifically indicated, we deduced based on raw data that this was Subject #16). 
+
+```{r echo = FALSE}
+# participant 16 deduced from SPSS
+df6_cric <- df6_cric %>%
+  filter(Subject!='16')
+
+df6_cric %>%
+   summarise(n_distinct(Subject))
+
+df6_spss <- df6_spss %>%
+  filter(subject != "16")
+
+df6_spss %>%
+  summarise(cleaned.sample.size = n())
+```
+
+> Time-rich participants rated the loss as more expensive when they thought about a small account (M = 8.31, 95% CI = [7.78, 8.84]) than when they thought about a large account (M = 6.50, 95% CI = [5.42, 7.58]),
+
+
+```{r echo = FALSE}
+
+df6_spss %>%
+  filter(slack == 1, large == 0) %>%
+  summarise(mean.expense.timerich.smallaccount = mean(expense), CI.low = mean(expense) - (1.96 * Sem(expense)), CI.high = mean(expense) + (1.96 * Sem(expense))) %>%
+  round(digits = 2)
+
+df6_spss %>%
+  filter(slack == 1, large == 1) %>%
+  summarise(mean.expense.timerich.largeaccount = mean(expense), CI.low = mean(expense) - (1.96 * Sem(expense)), CI.high = mean(expense) + (1.96 * Sem(expense))) %>%
+  round(digits = 2)
+```
+
+> whereas time-poor participants’ evaluations did not differ between the small-account condition (
```

Comment:
This was paper with many results, and you did an excellent job tracking down the possible sources of differences.  From your presentation, I understand that many of the extensions were making graphs.  That could be more clear in the write-up.